build.gradle

```diff
@@ -12,13 +12,14 @@ plugins {
   id 'application'
   id 'maven-publish'
   //id 'osgi' // discontinued - do we need it?
-  id 'com.bmuschko.docker-java-application' version '3.6.2'
+  id 'com.bmuschko.docker-java-application' version '9.3.1'
```

Comment:
What was the key thing to fix for using the new Docker plugin compared to version 6?

Reply:
The key thing was to use appropriate jre-slim image. For a moment when I saw in resource service that you used oracle java, I thought that we can use any java image and it will work. However, it appears that this isn't the case.

Reply:
Ok, so the problem was with the actual Docker build and not the Gradle plugin, I understand.